client/my-sites/site-settings/style.scss

```diff
@@ -233,6 +233,7 @@
 
 .site-settings__analytics-settings,
 .site-settings__general-settings,
+.site-settings__performance-settings,
```

Comment:
I believe this lived in the `SiteSettingsFormPerformance` that was deleted since then; I've added back the class on https://github.com/Automattic/wp-calypso/pull/29597/commits/d55d2188ce260326101690bc8dc94a1261b9f520